.gitignore

```diff
@@ -0,0 +1,56 @@
+# From https://github.com/github/gitignore
```

Comment:
Do we need all theses? Ex, what is the need for the build iOS and android folders

Reply:
We probably don't need all of these. I copied & pasted this file.